roles/ceilometer-common/meta/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+dependencies:
+  - role: monitoring-common
+  - role: logging-config
+    service: ceilometer
+    logdata: "{{ ceilometer.logs }}"
+  - role: openstack-source
+    project_name: ceilometer
+    project_rev: "{{ ceilometer.source.rev }}"
+    virtualenv: "{{ ceilometer.virtualenv }}"
+    python_requirements:
+      - { name: pymongo }
```

Comment:
We thought better not to pin the pymongo version since we don't pin the mongodb version (it will grow within version 3.0.x)